fuzzyfinder.go

```diff
@@ -144,22 +146,38 @@ func (f *finder) _draw() {
 			Bold(true)
 
 		// Add a space between '>' and runes.
-		f.term.SetContent(promptLinePad+w, height-1, r, nil, style)
+		f.term.SetContent(promptLinePad+w, maxHeight-1, r, nil, style)
 		w += runewidth.RuneWidth(r)
 	}
-	f.term.ShowCursor(promptLinePad+f.state.cursorX, height-1)
+	f.term.ShowCursor(promptLinePad+f.state.cursorX, maxHeight-1)
+
+	maxHeight -= 1
+
+	// Header line
+	if len(f.opt.header) > 0 {
+		w = 0
+		for _, r := range []rune(runewidth.Truncate(f.opt.header, maxWidth-3, "...")) {
```

Comment:
We use two dots to represent omitted string.
See: https://github.com/ktr0731/go-fuzzyfinder/blob/2bff5cea2b3a88d53c3c7401bafb0f1dd8347b70/fuzzyfinder.go#L229-L230

```suggestion
		for _, r := range []rune(runewidth.Truncate(f.opt.header, maxWidth-2, "..")) {
```

Reply:
I understand, thank you.